src/ConfigWidgets/SymbolsDialog.cpp

```diff
@@ -192,4 +194,14 @@ void SymbolsDialog::OnListItemSelectionChanged() {
   ui_->removeButton->setEnabled(!ui_->listWidget->selectedItems().isEmpty());
 }
 
+void SymbolsDialog::OnMoreInfoButtonClicked() {
+  QString url_as_string{
```

Comment:
I thought we had some restriction on linking to Stadia Docs, and there was a reason why we're not doing it anywhere. I might mix that up though - @ronaldfw ?

Reply:
Yeah, we generally don't want to do that, with possible exceptions (we link to the Stadia community sites for feedback). I'll share more internally. 

Apart from the feedback links, I think we do not want to have links to documentation that is not generally accessible for all open source users.